bin/leverj-market-maker-keeper

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+dir="$(dirname "$0")"/..
+source $dir/_virtualenv/bin/activate || exit
+export PYTHONPATH=$PYTHONPATH:$dir:$dir/lib/pymaker:$dir/lib/pyexchange:$dir/lib/ethgasstation-client:$dir/lib/gdax-client
```

Comment:
Why does leverj need `gdax-client` in `PYTHONPATH`?

Reply:
basically for price we eth-dai for live quoting which it gets from gdax